www/css-utils/card/readme.md

```diff
@@ -9,59 +9,49 @@ To apply card css utils, we can use 'fw-card-#{$elevation}'
 ## Sample - card with interactive component 
 ```html live
   <template>
-    <div>
-      <div class="fw-card-1 fw-p-24 fw-flex fw-flex-row">
-        <div class="fw-flex-grow">
-          <div class="fw-type-h5">Arabic</div>
-          <div class="fw-type-xs">Last updated - 25 June 2020</div>
-        </div>
-        <div class="fw-flex-grow-0">
-          <fw-button color="secondary" class="fw-type-h6"> Download existing </fw-button>
-          <fw-button color="secondary" class="fw-type-h6 fw-ml-8"> Update file </fw-button>
-        </div>
-        <div class="fw-flex-grow-0">
-          <fw-button size="icon" color="text" role="button" class="fw-ml-12">
-            <fw-icon name="delete" ></fw-icon>
-          </fw-button>
-        </div>
-      </div>
+    <div class="fw-card-1 fw-p-24 fw-flex fw-flex-row">
+      <section class="fw-flex-grow">
+        <h5 class="fw-type-h5 fw-my-0">Arabic</h5>
+        <p class="fw-type-xs fw-my-0">Last updated - 25 June 2020</p>
+      </section>
+      <section class="fw-flex-grow-0">
+        <fw-button color="secondary" class="fw-type-h6"> Download existing </fw-button>
+        <fw-button color="secondary" class="fw-type-h6 fw-ml-8"> Update file </fw-button>
+        <fw-button size="icon" color="text" role="button" class="fw-ml-12">
+          <fw-icon name="delete" ></fw-icon>
+        </fw-button>
+      </section>
     </div>
   </template>
 ```
 
 ## Sample - card with interactive component 
 ```html live
 <template>
-  <div>
-    <div class="fw-card-2 fw-p-24 fw-flex fw-flex-row fw-items-center">
-      <div class="fw-flex-grow">
-        <div class="fw-type-h6">Show typing indicator</div>
-        <div class="fw-type-xs">An indicator to see and share when messages are being typed</div>
-      </div>
-      <div class="fw-flex-grow-0">
-        <fw-toggle size="medium" checked></fw-toggle>
-      </div>
-    </div>
+  <div class="fw-card-2 fw-p-24 fw-flex fw-flex-row fw-items-center">
+    <section class="fw-flex-grow">
+      <h6 class="fw-type-h6 fw-my-0">Show typing indicator</h6>
+      <p class="fw-type-xs fw-my-0">An indicator to see and share when messages are being typed</p>
+    </section>
+    <section class="fw-flex-grow-0">
+      <fw-toggle size="medium" checked></fw-toggle>
+    </section>
   </div>
 </template>
 ```
 
 ## Sample - End user widget cards 
 ```html live
 <template>
-  <div>
-    <div class="fw-card-3 fw-p-20 fw-flex fw-flex-column">
-      <div class="fw-flex">
-        <span class="fw-flex-grow fw-type-h4">Chat with us</span>
-        <span class="fw-type-xs"><a href="#">View history</a></span>
-      </div>
-      <div>
-        <div class="fw-flex fw-flex-column fw-mt-4">
-          <div class="fw-type-h6">Support</div>
-          <div class="fw-type-xs">Typically replies within 5 minutes</div>
-        </div>
-      </div>
-    </div>
+  <div class="fw-card-3 fw-p-20 fw-flex fw-flex-column">
```

Comment:
can you also change it to section

Reply:
Same as https://github.com/freshworks/crayons/pull/629/files#r901594563.